projects/polaris/collection.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Isn't osc:workflows missing here?

Reply:
True and also osc:products right?

Reply:
We currently don't have osc:products in projects, so there's no link back from projects to products. We never had that in the past. Is that something that we want to add? Would quite a heavy change as we need to reprocess all the projects.

It probably makes sense, we just don't have it in any project yet and would need to add them to all projects with a script or so.